test/scripts/test_integrations.mjs

```diff
@@ -7,7 +7,6 @@ const integrations = await glob(resolve(__dirname, "../integration/*"));
 for (let dir of integrations) {
 	await within(async () => {
 		cd(dir);
-		// eslint-disable-next-line no-console
```

Comment:
I would like to keep on no-console as a rule if possible. Sometimes when i'm debugging i use a lot of console.logs and would love to catch it before it's submitted.